bitsail-connectors/bitsail-connectors-legacy/bitsail-connector-kafka/src/test/java/com/bytedance/bitsail/connector/legacy/kafka/common/KafkaPbMessage.java

```diff
@@ -0,0 +1,970 @@
+
```

Comment:
can we add the license header?

Reply:
This class is generated by protoc, which isn't dynamic. I removed it and made the test case directly read proto descriptor for job configuration.

Reply:
I think it's ok to put this class in our codebase, because if we generate this class from the test case, this class will be recognized by git and added to the git commit for those who run the test case. It's tricky to add this to `.gitignore` as well. Add the license header on an auto generated file is fine I believe.

Reply:
nvm, my concern didn't happen at all.